sdk/eventhub/event-hubs/samples/usingAadAuth.ts

```diff
@@ -27,7 +27,7 @@ import { EnvironmentCredential } from "@azure/identity";
 const evenHubsEndpoint = ""; // <your-eventhubs-namespace>.servicebus.windows.net
 const eventHubName = "";
 
-// Define AZURE_TENANT_ID, AZURE_CLIENT_ID and AZURE_CLIENT_SECRET of your AAD application in your environment
+// Define EVENTHUB_AAD_TENANT_ID, EVENTHUB_AAD_CLIENT_ID and EVENTHUB_AAD_CLIENT_SECRET of your AAD application in your environment
```

Comment:
Have you tested this sample after this change? I don't believe `EnvironmentCredential` will work as expected after this change.

The changes required from #4209 is for updating the env vars used in our tests, not the sample code or the usage of `EnvironmentCredential`

In this sample, we are telling our customers to use the `EnvironmentCredential` which depends on AZURE_TENANT_ID, AZURE_CLIENT_ID and AZURE_CLIENT_SECRET to be present. We cannot change this.

Reply:
Since this was just renaming of variables for consistency and not much change in implementation, I had run tests to verify the loading of env vars ( And not the samples or live tests (yet) )

Nothing in the code or issue surfaces that @azure/identity requires environment variables to be defined with specific names.
Looks like it's not possible to have SDK specific names as suggested at - https://github.com/Azure/azure-sdk-for-js/issues/4209#issuecomment-514313561
cc: @weshaggard

Reply:
https://www.npmjs.com/package/@azure/identity#environment-variables